CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
+<<<<<<< HEAD
 ## 0.4.1
 
 ### Common
 * Expose `package:turf/helpers.dart`.
+=======
+## 0.4.1 
+
+### Common
+* Bump platform Maps SDK dependencies to 10.10.0.
+* Fix issue with multiple maps overriding platform channels of the previous instances.
+>>>>>>> c718dda (Update changelog, bump platform Maps SDK version.)
```

Comment:
rm?

Reply:
Yep, hadn't added rebase fix to the commit